AdyenCard/Components/Card/CardComponentDelegate.swift

```diff
@@ -19,4 +19,9 @@ public protocol CardComponentDelegate: AnyObject {
     /// - Parameter value: Array of card types matching entered value. Null - if no data entered.
     /// - Parameter component: The `CardComponent` instance.
     func didChangeCardBrand(_ value: [CardBrand]?, component: CardComponent)
+
+    /// Called when shopper submits a card in `CardComponent`.
+    /// - Parameter value: Last 4 digits of card.
+    /// - Parameter component: The `CardComponent` instance.
+    func didSubmitLastFour(_ value: String, component: CardComponent)
```

Comment:
can we add a default implementation for the new delegate call back, since this is technically a breaking change.

merchants will get an error because the new call back isn't implemented, not to mention merchants might not need it.

Reply:
👍